src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -26,11 +26,24 @@
     private BlockNetworkUtil() {
     }
 
+    /**
+    * Get the side of a block that is rotated based on the defined side
+    *
+    * @param block - the block that is rotated
+    * @param definedSide - the definedSide to define which side of the rotated block to be returned
+    * @return a side rotation of the definedSide
```

Comment:
Again, confusing and doesn't actually give any extra info. You should also avoid using the name of the parameter in the description.